testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -153,6 +153,14 @@
             <artifactId>infinispan-server-hotrod</artifactId>
             <version>${infinispan.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-quarkus-server</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.ws</groupId>
```

Comment:
Could you please elaborate on why we need this dependency? Looks like you added after the first round of reviews?